workflow/r/barebones/v1/README.md

```diff
@@ -32,16 +32,22 @@ odbc                    1.3.3
 ### ODBC Setup
 - The `odbcinst.ini` file exists in the `/home/peak-user` directory, and the `ODBCSYSINI` env has been set accordingly.
 
+You can find more details about setting up ODBC [here](https://docs.aws.amazon.com/redshift/latest/mgmt/configure-odbc-connection.html).
```

Comment:
Hey aren't these only for redshift ? I guess we also need to add links for snowflake here ?